packages/blockchain-extension/extension/fabric/environments/AnsibleEnvironmentManager.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+import { ManagedAnsibleEnvironment } from './ManagedAnsibleEnvironment';
+
+export class AnsibleEnvironmentManager {
+
+    public static instance(): AnsibleEnvironmentManager {
+        return this._instance;
+    }
+
+    private static _instance: AnsibleEnvironmentManager = new AnsibleEnvironmentManager();
```

Comment:
shouldn't this be a `managedAnsibleEnvironmentManager`?

Reply:
Changed.